postgresql96/tests/test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+TESTDIR="$(dirname "${0}")"
+PLANDIR="$(dirname "${TESTDIR}")"
+
+SKIPBUILD=${SKIPBUILD:-0}
+
+hab pkg install --binlink core/bats
+
+hab pkg install core/busybox-static
+hab pkg binlink core/busybox-static ps
+hab pkg binlink core/busybox-static netstat
+hab pkg binlink core/busybox-static wc
+
+source "${PLANDIR}/plan.sh"
+
+if [ "${SKIPBUILD}" -eq 0 ]; then
+  set -e
+  pushd "${PLANDIR}" > /dev/null
+  build
+  source results/last_build.env
+  hab pkg install --binlink --force "results/${pkg_artifact}"
+  hab svc load "${pkg_ident}"
+  popd > /dev/null
+  set +e
+
+  # Give some time for the service to start up
+  sleep 5
+fi
+
+bats "${TESTDIR}/*.bats"
```

Comment:
You'll want to write this as `"${TESTDIR}"/*.bats` .  When the `*` is inside the quotes, it's treated as a string literal and isn't expanded.    

I *think* you don't even need the `*.bats`.  IIRC bats will automatically do that glob if passed a directory. 😸